dbc2val/test/test_example_mapping/test_example_mapping.py

```diff
@@ -76,55 +75,56 @@ def test_parking_lock():
     # CurrentGear tests
     vss_mapping = dbc_mapping[park_lock_index]
     assert vss_mapping.vss_name == "Vehicle.Powertrain.Transmission.IsParkLockEngaged"
-    assert isinstance(vss_mapping.transform,dict)
+    assert isinstance(vss_mapping.transform, dict)
     assert "mapping" in vss_mapping.transform
     mapping_list = vss_mapping.transform["mapping"]
     assert len(mapping_list) == 4
     # And now just check some items
     assert mapping_list[0]["from"] == "DI_GEAR_D"
-    assert mapping_list[0]["to"] == False
+    assert mapping_list[0]["to"] is False
     assert mapping_list[1]["from"] == "DI_GEAR_P"
-    assert mapping_list[1]["to"] == True
+    assert mapping_list[1]["to"] is True
 
     # Now test some transforms
     result = vss_mapping.transform_value("DI_GEAR_D")
-    assert result == False
+    assert result is False
     result = vss_mapping.transform_value("DI_GEAR_P")
-    assert result == True
-    
+    assert result is True
+
     # For non successful mapping old value shall be kept and None returned
     result = vss_mapping.transform_value("Something undefined")
-    assert result == None
+    assert result is None
+
 
 def test_vehicle_speed():
     dbc_mapping = mapper["DI_uiSpeed"]
     assert len(dbc_mapping) == 2
-    
+
     if dbc_mapping[0].vss_name == "Vehicle.Speed":
         speed_index = 0
     else:
         speed_index = 1
     vss_mapping = dbc_mapping[speed_index]
     assert vss_mapping.vss_name == "Vehicle.Speed"
-    assert vss_mapping.transform == None
-    
+    assert vss_mapping.transform is None
+
     # Now test some transforms
     result = vss_mapping.transform_value(25.3)
     assert result == 25.3
 
+
 def test_vehicle_obd_speed():
     dbc_mapping = mapper["DI_uiSpeed"]
     assert len(dbc_mapping) == 2
-    
+
     if dbc_mapping[0].vss_name == "Vehicle.OBD.Speed":
         speed_index = 0
     else:
         speed_index = 1
     vss_mapping = dbc_mapping[speed_index]
     assert vss_mapping.vss_name == "Vehicle.OBD.Speed"
-    assert vss_mapping.transform == None
-    
+    assert vss_mapping.transform is None
+
     # Now test some transforms
     result = vss_mapping.transform_value(25.3)
     assert result == 25.3
-    
```

Comment:
no new line at end of file intended?

Reply:
When looking in an editor there seems to be a newline. What previously existed was a number of blanks on last line and no end-of-line after the blanks. Those blanks have been removed, and this seems to be how github presents it. So the resulting file seems to me to be corrected.